packages/botonic-nlp/src/encode/indexed-items.ts

```diff
@@ -26,6 +27,12 @@ export class IndexedItems {
     return this.items[idx]
   }
 
+  maskUnknownItems(sequence: string[], unknownToken = UNKNOWN_TOKEN): string[] {
```

Comment:
I prefer not to define any default value for unknownToken to avoid dependencies from encode to preprocess folder (to try to avoid circular dependencies)
rename unknownToken to unknownItem

Reply:
makes sense! 👍🏽